tests/model_executor/test_guided_processors.py

```diff
@@ -205,24 +203,3 @@ def test_guided_decoding_backend_options():
 
     no_fallback = GuidedDecodingParams(backend="xgrammar:option-1,no-fallback")
     assert no_fallback.no_fallback()
-
-
-def test_pickle_xgrammar_tokenizer_data():
```

Comment:
why dont we want to these this anymore?

Reply:
Because most of the value in TokenizerData are now serialisable.

but I will readd a tests to make sure that will always be the case.